src/games/supported/Tetris.hpp

```diff
@@ -0,0 +1,80 @@
+/* *****************************************************************************
+ * The lines 68 and 76 are based on Xitari's code, from Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License version 2
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ * *****************************************************************************
+ * A.L.E (Arcade Learning Environment)
+ * Copyright (c) 2009-2013 by Yavar Naddaf, Joel Veness, Marc G. Bellemare and 
+ *   the Reinforcement Learning and Artificial Intelligence Laboratory
+ * Released under the GNU General Public License; see License.txt for details. 
+ *
+ * Based on: Stella  --  "An Atari 2600 VCS Emulator"
+ * Copyright (c) 1995-2007 by Bradford W. Mott and the Stella team
+ *
+ * *****************************************************************************
+ */
+#ifndef __TETRIS_HPP__
+#define __TETRIS_HPP__
+
+#include "../RomSettings.hpp"
+
+
+/* RL wrapper for Tetris */
+class TetrisSettings : public RomSettings {
+
+    public:
+
+        TetrisSettings();
+
+        // reset
+        void reset();
+
+        // is end of game
+        bool isTerminal() const;
+
+        // get the most recently observed reward
+        reward_t getReward() const;
+
+        // the rom-name
+        const char* rom() const { return "tetris"; }

```

Comment:
P.S. The original Tetris26.BIN file from 1997 did not randomize the sequence a block which made the game less attractive for ML. I've [added](https://github.com/udibr/tetris26) a pesudo random and along the way decided to rename this to tetris.bin.

Please move comments on Tetris to a sperate [pull request](https://github.com/mgbellemare/Arcade-Learning-Environment/pull/93) I created on the dev branch